### PR TITLE
switches Pulp 3 to dynaconf

### PR DESCRIPTION
**Pull request by dkliban**

### CODE REVIEW

File: ansible/roles/dev/files/django.bashrc [hunk 1]
```diff
@@ -1 +1,3 @@
 export DJANGO_SETTINGS_MODULE=pulpcore.app.settings
+export PULP_DEBUG=True
+export PULP_SECRET_KEY=$(cat /dev/urandom | tr -dc 'a-z0-9!@#$%^&*(\-_=+)' | head -c 50)
```
Comment (dparalen) [inline ansible/roles/dev/files/django.bashrc]: wouldn't this change every time you logged in? Also mind the missing new line ;)

**Comment by dkliban** [comment]
@dparalen Thanks for the review! I updated it to only generate the key once.

**Comment by dparalen** [comment]
@dkliban thanks for the update!
I'll install deploy with the change briefly (in the middle of debugging another devel issue :)

File: ansible/roles/dev/tasks/main.yml [hunk 2]
```diff
@@ -138,16 +138,16 @@
   - name: Install server.yaml Config file
```
Comment (dparalen) [inline ansible/roles/dev/tasks/main.yml]: Nit(-ish): would you mind s/Install server .yaml Config file/Install server Dynaconf file/ or so?

**Pull request merged**